src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -145,6 +145,26 @@ paths:
             trace.
       tags:
       - Auth
+  /api/auth/token:
+    get:
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/vnd.ceph.api.v1.0+json:
+              type: object
+          description: OK
+        '400':
+          description: Operation exception. Please check the response body for details.
+        '401':
+          description: Unauthenticated access. Please login first.
+        '403':
+          description: Unauthorized access. Please check your permissions.
+        '500':
+          description: Unexpected error. Please check the response body for the stack
+            trace.
+      tags:
+      - Auth
```

Comment:
This spec probably needs to be refreshed.

Reply:
Yes no need for it, I'm going to remove it too!